### PR TITLE
Add documentation and test commands for the tput helper

### PR DESCRIPTION
**Pull request by jubianchi**
See #7

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -356,3 +357,25 @@ $app
             );
         });
 ```
+
+#### Tput
+
+The tput helper will help you get informed about user's terminal capabilities. The helper provides
+a single entry point to all capabilities: the `get` method. Here is how you would do to get the `clear_screen`
+capability:
```
Comment (Hywan) [inline README.md]: No `count` or `has` methods?

**Comment by Hywan** [comment]
:+1: else.

File: README.md (hunk 1 above)
Comment (jubianchi) [inline README.md]: The `get` method falls back on ˋcount` or ˋhas`. This was done to make the API simple. 
The user can still use native methods: the helper extends the original Tpit class.

Comment (Hywan) [inline README.md]: Ok.

**Pull request merged**